src/cow-react/modules/swap/pure/Row/RowFeeContent/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { ThemedText } from 'theme'
+
+import { RowBetween, RowFixed } from 'components/Row'
+import { MouseoverTooltipContent } from 'components/Tooltip'
+import { StyledInfo } from '@cow/pages/Swap/styleds'
+import { LightGreyText } from '@cow/modules/swap/pure/styled'
+import { RowFeeProps } from '@cow/modules/swap/containers/RowFee'
+import { RowCommonProps } from '../typings'
+
+export type RowFeeAuxProps = RowCommonProps & {
+  includeGasMessage: string
+  tooltip: string
+  feeToken: string
+  feeUsd?: string
+  fullDisplayFee: string
+  feeCurrencySymbol: string
+}
+
+export function RowFeeContent(props: Omit<RowFeeProps, 'feeFiatValue' | 'allowsOffchainSigning'> & RowFeeAuxProps) {
```

Comment:
The type of props looks too complex, better to copy props into a new interface.
p.s. https://en.wikipedia.org/wiki/Interface_segregation_principle